Kitodo/src/main/java/org/kitodo/production/forms/createprocess/ProcessDataTab.java

```diff
@@ -192,23 +200,85 @@ public void generateProcessTitleAndTiffHeader() {
         Process process = this.createProcessForm.getMainProcess();
         try {
             StructuralElementViewInterface docTypeView = createProcessForm.getRulesetManagement().getStructuralElementView(
-                docType, createProcessForm.getAcquisitionStage(), createProcessForm.getPriorityList());
+                    docType, createProcessForm.getAcquisitionStage(), createProcessForm.getPriorityList());
             String processTitle = docTypeView.getProcessTitle().orElse("");
             if (processTitle.isEmpty()) {
                 Helper.setErrorMessage("newProcess.titleGeneration.creationRuleNotFound",
                     new Object[] {getDocTypeLabel(docType), process.getRuleset().getTitle() });
             }
-            this.atstsl = ProcessService.generateProcessTitle(this.atstsl, processDetails,
-                processTitle, process);
+
+            String currentTitle = TitleGenerator.getValueOfMetadataID(TitleGenerator.TITLE_DOC_MAIN, processDetails);
+
+            if (StringUtils.isBlank(currentTitle)) {
+                Process parentProcess = createProcessForm.getTitleRecordLinkTab().getTitleRecordProcess();
+                if (Objects.nonNull(parentProcess)) {
+                    currentTitle = getTitleFromMetaXML(parentProcess);
+                } else {
+                    currentTitle = getTitleFromAncestors();
+                }
+            }
+
+            String atstsl = ProcessService.generateProcessTitleAndGetAtstsl(processDetails, processTitle, process,
+                currentTitle);
+
             // document name is generally equal to process title
             this.tiffHeaderDocumentName = process.getTitle();
-            this.tiffHeaderImageDescription = ProcessService.generateTiffHeader(
-                    processDetails, this.atstsl, ServiceManager.getImportService().getTiffDefinition(), this.docType);
+            this.tiffHeaderImageDescription = ProcessService.generateTiffHeader(processDetails, atstsl,
+                ServiceManager.getImportService().getTiffDefinition(), this.docType);
         } catch (ProcessGenerationException e) {
             Helper.setErrorMessage(e.getLocalizedMessage(), logger, e);
         }
         Ajax.update("editForm:processFromTemplateTabView:processDataEditGrid",
-                "editForm:processFromTemplateTabView:processMetadata");
+            "editForm:processFromTemplateTabView:processMetadata");
+    }
+
+    private String getTitleFromMetaXML(Process process) {
+        String xpath = "//kitodo:metadata[@name='" + TitleGenerator.TITLE_DOC_MAIN + "']";
+        try {
+            NodeList nodeList = ServiceManager.getProcessService().getNodeListFromMetadataFile(process, xpath);
+            if (nodeList.getLength() > 0) {
+                return nodeList.item(0).getTextContent();
```

Comment:
This assumes that only _one_ `TitleDocMain` metadata can occur in a metadata file, but that is - at least technically - not neccessary always the case (if the ruleset is configured accordingly, chapters could have `TitleDocMain` metadata as well, for example)

Reply:
@solth What exactly are you proposing here?  

Should the XPath be defined more clearly, or is there a TitleDocMain priority list that can be processed if several potential fields exist?

Do you have PPN's to check the different cases?

Reply:
Title should be taken from the `<key use="title">` defined in the ruleset